src/www/app/cordova_main.ts

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/// <reference path='../../types/ambient/webintents.d.ts'/>
 /// <reference types='cordova'/>
+/// <reference path='../../types/ambient/webintents.d.ts'/>
```

Comment:
what caused you to move this down?

Reply:
I did it. In general we prefer to place the local libraries in the end, and the cordova lib is also imported first.